codalib/util.py

```diff
@@ -10,14 +10,15 @@
     import simplejson as json
 from lxml import etree
 import uuid
-from datetime import datetime, tzinfo, timedelta
 from . import bagatom
+from .xsdatetime import xsDateTime_format
+from datetime import datetime
+import time
```

Comment:
While we are doing stuff with imports, can you replace all that simplejson/json stuff just with `import json`, since I don't think we are supporting <2.6, and also reorganize the imports per https://www.python.org/dev/peps/pep-0008/#imports.